packages/frontend/tailwind.config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         600: '#848484',
         650: '#5C5C5C',
         700: '#565656',
+        750: '#424045',
         800: '#424242',
```

Comment:
Are you sure we need that? It's okay to change the color slightly if we have a similar in the codebase already.